src/translations/en.json

```diff
@@ -968,7 +968,7 @@
         "logs": {
           "caption": "Logs",
           "description": "View the Home Assistant logs",
-          "details": "Log Details ({level})",
```

Comment:
This will break all current translations, we cant remove variables from current values.

Reply:
The problem is that with the placeholder we end up with those "ugly" blanks for it:

![image](https://user-images.githubusercontent.com/114137/109042145-23278600-76d0-11eb-8145-4e7f21038b30.png)

Is there a trick to prevent localize() from adding those?

Reply:
They are in the german translation, it is not caused by `localize` 😄 

<https://github.com/home-assistant/frontend/blob/5dfe3b68e67190da4224dcaf533e3a2a8f0b9f41/translations/frontend/de.json#L1995>